ios/MattermostShare/SessionManager.m

```diff
@@ -37,6 +37,7 @@ -(void)URLSession:(NSURLSession *)session task:(NSURLSessionDataTask *)task didC
     NSLog(@"ERROR %@", [error userInfo]);
     NSLog(@"invalidating session %@", requestWithGroup);
     [session invalidateAndCancel];
+    self.sendShareEvent(@"extensionPostFailed");
```

Comment:
Curious why not a `self.failExtension`, with the implementation sending a hard-coded `@"extensionPostFailed"` in one place vs. lots of places, especially given that there is but one event from this object right now.

Reply:
yeah I just made that block more general cause I'm thinking of sending back events that track upload progress in the future